src/overlays/actors/ovl_Bg_Keikoku_Saku/z_bg_keikoku_saku.c

```diff
@@ -28,21 +32,70 @@ const ActorInit Bg_Keikoku_Saku_InitVars = {
     (ActorFunc)BgKeikokuSaku_Draw,
 };
 
-#endif
+void BgKeikokuSaku_Init(Actor* thisx, GlobalContext* globalCtx) {
+    BgKeikokuSaku* this = THIS;
+    s32 pad;
+    CollisionHeader* colHeader;
+
+    colHeader = NULL;
+    DynaPolyActor_Init(&this->Dyna, 0);
+    CollisionHeader_GetVirtual(&object_keikoku_obj_Colheader_002300, &colHeader);
+    this->Dyna.bgId = DynaPoly_SetBgActor(globalCtx, &globalCtx->colCtx.dyna, &this->Dyna.actor, colHeader);
+    this->unk160 = BGKEIKOKUSAKU_GET_SWITCHFLAG(thisx);
+    if (Flags_GetSwitch(globalCtx, this->unk160)) {
+        this->Dyna.actor.world.pos.z = 2659.0f;
+        return;
+    }
+    this->actionFunc = func_80A5389C;
```

Comment:
```suggestion
    if (Flags_GetSwitch(globalCtx, this->unk160)) {
        this->Dyna.actor.world.pos.z = 2659.0f;
    } else {    
        this->actionFunc = func_80A5389C;
    }
```
Would prefer an `if...else` rather than early return.